dvc/repo/experiments/init.py

```diff
@@ -200,21 +199,19 @@ def init(
     defaults = defaults.copy() if defaults else {}
     overrides = overrides.copy() if overrides else {}
 
-    with_live = type == "dl"
+    with_live = "live" in overrides
 
     if interactive:
         defaults = init_interactive(
             validator=partial(validate_prompts, repo),
             defaults=defaults,
-            live=with_live,
             provided=overrides,
+            live=with_live,
```

Comment:
Let's drop `live` kwarg, since it depends on `overrides`/`provided` values from CLI rather than the mode/type.

You can replace with something like following in https://github.com/iterative/dvc/blob/4b6f2306faaa051f595e56e216e1d5b4f1a1e3c1/dvc/repo/experiments/init.py#L86-L89
```python
output_keys = ["models"]
if "live" not in provided:
    output_keys.extend(["metrics", "plots"])
outputs_prompts = lremove(provided.keys(), output_keys)
```